src/frontend/Sider/Behandling/Vilkårvurdering/EndreVilkår.tsx

```diff
@@ -214,31 +218,65 @@ export const EndreVilkår: FC<EndreVilkårProps> = (props) => {
             <FeilmeldingMaksBredde $maxWidth={152}>
-                <MonthInput
-                    label="Fra"
-                    size="small"
-                    value={fom}
-                    feil={feilmeldinger.fom}
-                    readOnly={!props.alleFelterKanRedigeres}
-                    onChange={(dato) => {
-                        settFom(dato ? tilFørsteDagenIMåneden(dato) : undefined);
-                        settDetFinnesUlagredeEndringer(true);
-                        settFeilmeldinger((prevState) => ({ ...prevState, fom: undefined }));
-                    }}
-                />
+                {skalBrukeMånedÅrVelger ? (
+                    <MånedÅrVelger
+                        label="Fra"
+                        size="small"
+                        årMånedInitiell={fom}
+                        feilmelding={feilmeldinger.fom}
+                        lesevisning={!props.alleFelterKanRedigeres}
+                        onEndret={(dato) => {
+                            settFom(dato ? tilFørsteDagenIMåneden(dato) : undefined);
+                            settDetFinnesUlagredeEndringer(true);
+                            settFeilmeldinger((prevState) => ({ ...prevState, fom: undefined }));
+                        }}
+                        antallÅrFrem={3}
+                        antallÅrTilbake={3}
+                    />
+                ) : (
+                    <MonthInput
+                        label="Fra"
+                        size="small"
+                        value={fom}
+                        feil={feilmeldinger.fom}
+                        readOnly={!props.alleFelterKanRedigeres}
+                        onChange={(dato) => {
+                            settFom(dato ? tilFørsteDagenIMåneden(dato) : undefined);
+                            settDetFinnesUlagredeEndringer(true);
+                            settFeilmeldinger((prevState) => ({ ...prevState, fom: undefined }));
+                        }}
+                    />
+                )}
             </FeilmeldingMaksBredde>
             <FeilmeldingMaksBredde $maxWidth={152}>
-                <MonthInput
-                    label="Til"
-                    size="small"
-                    value={tom}
-                    feil={feilmeldinger.tom}
-                    onChange={(dato) => {
-                        settTom(dato ? tilSisteDagenIMåneden(dato) : undefined);
-                        settDetFinnesUlagredeEndringer(true);
-                        settFeilmeldinger((prevState) => ({ ...prevState, tom: undefined }));
-                    }}
-                />
+                {skalBrukeMånedÅrVelger ? (
+                    <MånedÅrVelger
+                        label="Til"
+                        size="small"
+                        årMånedInitiell={tom}
+                        feilmelding={feilmeldinger.tom}
+                        lesevisning={!props.alleFelterKanRedigeres}
```

Comment:
```suggestion
                        lesevisning={props.helePeriodenErLåstForRedigering}
```
(denne må også sendes inn der komponenten taes i bruk)